packages/core/src/VRange.ts

```diff
@@ -76,6 +76,10 @@ export class VRange {
         if (boundaryPoints) {
             const [start, end] = boundaryPoints;
             const [startNode, startPosition] = start;
+
+            // Prevent setting a collapsed range into a not-editable node.
+            if (start === end && !startNode.editable) return;
```

Comment:
But a range does not necessarily comes from the selection of the editor.
A range can be created from a command, wich does not care about the selection.
And that's another argument to have the range having a `mode` rather than the `editor`.

Reply:
Well this was an extra safety as it _is_ handled in the `EventNormalizer` so no problem.